Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -29,7 +29,9 @@
         </mat-list-item>
         <mat-list-item>
           <button mat-menu-item (click)="navigateToOwnProfile(); sidenav.close()">
-            <mat-icon>perm_identity</mat-icon>
+            <mat-icon *ngIf="currentUser == null">perm_identity</mat-icon>
+            <app-profile-picture *ngIf="currentUser != null" [imageSize]="50" class="pb-self-centered circle square-img36 mobileedition" [user]="currentUser">

```

Comment:
why is this a different size than the other icon? The material icon size does not change so I think both should have the same size.

Reply:
This styling is for the desktop menu bar without icons. In this context a larger user image (36px) seems appropriate and is also used by other apps, like azure devops, ...